### ISSUE TITLE
Getting Back on Track

### ISSUE THREAD

**Issue by notnola** [notnola/pinychat#42]
@swiftSwathSee, over the next few days lets focus on creating and maintaining seperate and individual identities of Pinychat and any code that uses it. Pinychat's main goal is to provide a set of Python functions/methods to enable the creation of other tools. Example - A room helper bot, TriviaBot, user locator, etc. And it seems to be slowly wandering off course of that goal.
Pinychat's mission was never intended to be a full fledged terminal client for Timychat. However I am willing to support that effort in any way possible. I would love to have a well documented unoffical Tinychat Python API to provide to the community.

I will be assisting you in this endeavor as soon as possible, probably starting Feb24. I encourage any other developers using Pinychat to please help out by submitting pull requests relating to this topic. 

-Regards,
---Nola

**Comment by swiftSwathSee** [comment]
Agreed, module support is definitely very much needed. I've kept the existing code structure intact with almost all of my commits, so hopefully from your perspective not much has changed since your hiatus. There is of course some stuff that will likely have to be moved around (like the "init" and "settings" sections), but everything else should be the same for you.

One caveat is that despite how much I've worked on this project, I'm actually not 100% familiar with its overall code structure. That's why things like colorama and the section with `ignoreList = []` are just shoved into the main section, rather than something more modular like a class. So for now I might not be very useful when it comes to turning the whole project into a module.
